index.html

```diff
@@ -1470,28 +1472,25 @@
       <p>
         The <dfn>NFCWatchOptions.kind</dfn> property
         denotes the string value which is used for matching the
-        <code>kind</code> property of each <code>NFCRecord</code> object
-        in a <a>Web NFC message</a>.
+        <code>
+        <a href="#idl-def-nfcrecordtype">kind</a></code> property of each
+        <code><a>NFCRecord</a></code> object in a <a>Web NFC message</a>.
         The default value <code>""</code> means that no matching happens.
       </p>
       <p>
         The <dfn>NFCWatchOptions.type</dfn> property
         denotes the <a>match pattern</a> which is used for matching the
-        <code>type</code> property of each <code>NFCRecord</code> object
-        in a <a>Web NFC message</a>.
+        <code><a href="#dom-nfcrecord-type">type</a></code> property of each
```

Comment:
`<a>NFCRecord.type</a>` might auto-link and give you code formatting by default.